packages/web/src/components/modal-drawer/ModalDrawer.tsx

```diff
@@ -7,15 +15,21 @@ import { useIsMobile } from 'hooks/useIsMobile'
 import styles from './ModalDrawer.module.css'
 
 type ModalDrawerProps = ModalProps &
-  DrawerProps & { useGradientTitle?: boolean }
+  DrawerProps & {
+    useGradientTitle?: boolean
+    newModal?: boolean
```

Comment:
can we have a doc string here. it's not immediately obvious what this prop is for

Reply:
yeah totes :)